### ISSUE TITLE
Use configuration file instead of so many variables

### ISSUE THREAD

**Issue by bumblebeen** [apigee/apigeetool-node#106]
Hi,

I would like to ask for a feature request.
Since it is too tedious to type everything in a single command like -o org -e environment
Also it is hard to maintain.
Can there be a file like '.apigee' where apigeetool node reads file for the options?
You may refer to AWS-cli, it can have many profiles with different environment setup.

**Comment by AdamMagaluk** [comment]
Couldn't agree more. 

There is a `netrc` file that is available for username and password if that would help you with a few less parameters. See #45 we still need to document this better and expand to other flags like `org` and `env`.

**Comment by ivan-novakov** [comment]
Right, I agree about the config file, but I would better have it specified on the command line:
```
apigeetool <commmand> --config ~/.apigeetool.cfg
```

**Comment by sveyret** [comment]
Hi,
When I saw this immense list of options, I had exactly the same feeling. I've just released the library [confinode](https://www.npmjs.com/package/confinode) which purpose is to manage configuration files. This would probably help you add a configuration file in this project.
Please ask me if you have questions or need help on this.
